src/shared/entity.model.ts

```diff
@@ -25,10 +24,10 @@ export default abstract class Model extends BaseEntity {
   @UpdateDateColumn({ type: 'timestamptz' })
   updated_at: Date;
 
-  @BeforeInsert()
-  createUuid() {
-    this.uuid = uuid();
-  }
+  // @BeforeInsert()
+  // createUuid() {
+  //   this.uuid = uuid();
+  // }
```

Comment:
I think no need to these comments, so they can be removed.

Reply:
done